Navaids/FIXES_UK.txt

```diff
@@ -865,6 +866,7 @@ TORGO N053.07.57.000 W003.15.59.000
 TOSVA N051.40.37.000 E002.04.26.000
 TOTRI N051.46.30.000 E000.11.48.000
 TOVGU N051.44.03.000 E000.45.38.000
+TOXTA N055.15.06.000 W004.27.05.000
```

Comment:
on airway or only for procedure? In ENR 4.4?

Reply:
It's in ENR 4.4, yes ![ENR 4.4](https://i.imgur.com/2UkwDET.png)